src/main/java/com/spotify/docker/BuildMojo.java

```diff
@@ -779,6 +787,10 @@ public static String separatorsToUnix(final String path) {
     if (noCache) {
       buildParams.add(DockerClient.BuildParam.noCache());
     }
+    if (!buildArgs.isEmpty()) {
+      buildParams.add(DockerClient.BuildParam.create("buildargs", 
+        URLEncoder.encode(new ObjectMapper().writeValueAsString(buildArgs), "UTF-8")));
```

Comment:
it would be better to store the ObjectMapper as a field of the class, to avoid re-instantiation.